skfda/representation/_functional_data.py

```diff
@@ -1163,22 +1163,25 @@ def take(  # noqa: WPS238
         fill_value: Optional[T] = None,
         axis: int = 0,
     ) -> T:
-        """Take elements from an array.
+        """
+        Take elements from an array.
 
         Parameters:
-            indices:
-                Indices to be taken.
+            indices: Indices to be taken.
             allow_fill: How to handle negative values in `indices`.
-                * False: negative values in `indices` indicate positional
-                  indices from the right (the default). This is similar to
-                  :func:`numpy.take`.
-                * True: negative values in `indices` indicate
-                  missing values. These values are set to `fill_value`. Any
-                  other negative values raise a ``ValueError``.
+            * False: negative values in `indices` indicate positional
```

Comment:
This is wrongly represented as an additional parent in the generated documentation.